api/v1/sriovnetworknodestate_types.go

```diff
@@ -87,6 +88,7 @@ type VirtualFunction struct {
 // SriovNetworkNodeStateStatus defines the observed state of SriovNetworkNodeState
 type SriovNetworkNodeStateStatus struct {
 	Interfaces    InterfaceExts `json:"interfaces,omitempty"`
+	RdmaMode      string        `json:"rdmaMode,omitempty"`
```

Comment:
we need to talk about this one in the community I think we should have an higher level object first something like System/Host and we have that inside

Reply:
done